Packages/Core/Editor/Generator/Generator.cs

```diff
@@ -257,7 +236,7 @@ public static List<string> CreateTemplateConditions(bool isValueTypeEquatable, s
         /// </code>
         /// </example>
         [Obsolete("Use the same method with a Type parameter instead.", false)]
-        public static void Generate(AtomReceipe atomReceipe, string baseWritePath, string[] templatePaths, List<string> templateConditions, Dictionary<string, string> templateVariables)
+        public static void Generate(AtomRecipe atomReceipe, string baseWritePath, string[] templatePaths, List<string> templateConditions, Dictionary<string, string> templateVariables)
         {
             var (atomType, valueType) = atomReceipe;
```

Comment:
Receipe still lurks on this line... 😄

Reply:
Nice catch, I've fixed it locally so it will come around one of these updates.